.husky/pre-commit

```diff
@@ -12,7 +12,9 @@ if git diff-index --cached HEAD | tr '\t' ' ' | sed 's/^.*[ ]//' | grep '^cli/'
 	read res < /dev/tty
 	case $res in
 		""|Y|y)
-			bun run changeset < /dev/tty
+			if bun run changeset < /dev/tty; then
+				git add .changeset
```

Comment:
On second thought... should this be something like `git add .changeset/*.md`?

I would need to be careful about making sure the glob works within the shell (who's handling it, the shell or git)...

The current solution is mostly fine, except if one were to intentionally leave changes in the `.changeset/` directory that you do *not* want to commit, it is not going to work if you are committing something in the `cli/` directory *and* make a changeset using the pre-commit hook.